parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -810,16 +817,12 @@ oras_login_with_kubelet_identity() {
 
     set +x 
     access_url="http://169.254.169.254/metadata/identity/oauth2/token?api-version=2018-02-01&resource=https://management.azure.com/&client_id=$client_id"
-    raw_access_token=$(retrycmd_get_access_token_for_oras 10 5 $access_url)
+    raw_access_token=$(retrycmd_get_access_token_for_oras 5 15 $access_url)
     ret_code=$? 
-    if [ $ret_code -ne 0 ]; then
-        echo "failed to retrieve access token: $ret_code"
+    if [ $ret_code -ne 0 ] || [[ "$raw_access_token" == *"error"* ]]; then
```

Comment:
Could you add a comment explaining the expected format of the output here and why we need to check both return code and contents of $raw_access_token?

Reply:
it is not necessary, the second condition is only to keep the old logic to make the pr change as small as possible

Reply:
removed